config/README.rst

```diff
@@ -148,6 +148,11 @@ The following options are only needed one time as their values re stored in
 --with-sphinx-doctests
   Enable running the documentation as doctest using Sphinx.
 
+--no-flake8
+  Don't add ``flake8`` and ``isort`` linting steps to the configuration. If
```

Comment:
The option does more then the name implies. But I am not able to come up with an alternative name, so I think it is okay.

Reply:
I like it better than `--no-lint`, given that we still have `tox -e lint` that does some checking.